### PR TITLE
feat(words): add words related to Qt and OpenCV

### PR DESCRIPTION
**Pull request by h-ohta**
fourcc:
- https://docs.opencv.org/3.4/dd/d9e/classcv_1_1VideoWriter.html#afec93f94dc6c0b3e28f4dd153bc5a7f0
- is used in https://github.com/autowarefoundation/autoware.universe/blob/ce8f2c36122f888ec2bc47085639fb1e77166f2b/common/tier4_screen_capture_rviz_plugin/src/screen_capture_panel.cpp#L133

qobject:
 - https://doc.qt.io/qt-6/qobject.html
 - is used in https://github.com/autowarefoundation/autoware.universe/blob/ce8f2c36122f888ec2bc47085639fb1e77166f2b/common/tier4_screen_capture_rviz_plugin/src/screen_capture_panel.cpp#L115

### CODE REVIEW

**Comment by kenji-miyake** [comment]
Please fix the PR title.

**Comment by h-ohta** [comment]
@kenji-miyake Sorry I fixed.

File: .cspell.json [hunk 1]
```diff
@@ -1128,7 +1129,9 @@
     "pyyaml",
     "qdisc",
     "qframe",
+    "QImage",
     "qlen",
+    "QObject",
```
Comment (kenji-miyake) [inline .cspell.json:1134]: I believe you should change the variable names. `QImage` and `QObject` don't cause errors.
image

```suggestion
    "qlen",
```

Comment (h-ohta) [reply .cspell.json:1134]: Oh...
I'll delete qimage and fix variables.
However, qobject is included at `qobject_cast`, so I'll fix QObject to qobject

**Review by kenji-miyake — APPROVED** [review]

**Pull request merged**